build/packaging/rpm/condor.spec

```
@@ -1551,7 +1551,9 @@ fi
 test -x /usr/sbin/selinuxenabled && /usr/sbin/selinuxenabled
 if [ $? = 0 ]; then
    /usr/sbin/semodule -i /usr/share/condor/htcondor.pp
+%if 0%{?rhel} >= 9
```

```suggestion
%if 0%{?rhel} < 9
```
I think you _don't_ want to run this on EL9.

---

D'oh